tmk_core/protocol.mk

```diff
@@ -50,5 +50,9 @@ ifdef ADB_MOUSE_ENABLE
 	 OPT_DEFS += -DADB_MOUSE_ENABLE -DMOUSE_ENABLE
 endif
 
+ifdef XT_ENABLE
+  SRC += $(PROTOCOL_DIR)/xt_interrupt.c
```

Comment:
Add `OPT_DEFS += -DXT_ENABLE` here as well, so this can be detected by c/h files.